workspace.json

```diff
@@ -1,8 +1,6 @@
 {
   "version": 2,
   "projects": {
-    "data-mapper": "libs/data-mapper",
-    "data-mapper-standalone": "apps/data-mapper-standalone",
```

Comment:
What is this doing?

Reply:
Only for testing, forgot to revert it before the PR